<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgModificaDocentes</class>
 <widget class="QDialog" name="dlgModificaDocentes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>557</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>692</width>
    <height>557</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>557</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modificacion de Docentes</string>
  </property>
  <widget class="QLabel" name="lblDNI">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DNI</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblApellido">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDomicilio">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Domicilio</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLocalidad">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Localidad</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCodigoPostal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Código Postal</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFechaIngreso">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha de Ingreso</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEMail">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>E-Mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDNI">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leApellido">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDomicilio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLocalidad">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCodigoPostal">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEMail">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="twDocentes">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>DNI</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lblBusqueda">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Búsqueda</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leBusqueda">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="twTelefonos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string>Nro Teléfono</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Aclaración</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="twMateriasElegidas">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string>Materias dictadas</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pbAgregarMateria">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGuardarCambios">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>480</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar Cambios</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBorrarDocente">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar Docente</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deFechaIngreso">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEliminarTelefono">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Seleccionado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDesvincularMateria">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twMateriasAElegir">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string>Materias Elegir</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="gbNuevoTelefono">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Nuevo Telefono</string>
   </property>
   <widget class="QPushButton" name="pbAgregarTelefono">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar
Teléfono</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTelefono">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTelefono">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAclaracion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblAclaracion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aclaración</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leBusqueda</tabstop>
  <tabstop>twDocentes</tabstop>
  <tabstop>leDNI</tabstop>
  <tabstop>deFechaIngreso</tabstop>
  <tabstop>leApellido</tabstop>
  <tabstop>leNombre</tabstop>
  <tabstop>leDomicilio</tabstop>
  <tabstop>leLocalidad</tabstop>
  <tabstop>leCodigoPostal</tabstop>
  <tabstop>leEMail</tabstop>
  <tabstop>leTelefono</tabstop>
  <tabstop>leAclaracion</tabstop>
  <tabstop>pbAgregarTelefono</tabstop>
  <tabstop>twMateriasAElegir</tabstop>
  <tabstop>twMateriasElegidas</tabstop>
  <tabstop>twTelefonos</tabstop>
  <tabstop>pbAgregarMateria</tabstop>
  <tabstop>pbDesvincularMateria</tabstop>
  <tabstop>pbEliminarTelefono</tabstop>
  <tabstop>pbBorrarDocente</tabstop>
  <tabstop>pbGuardarCambios</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
